challenge-045/markus-holzer/raku/ch-2.raku

```diff
@@ -0,0 +1 @@
+$*PROGRAM-NAME.IO.slurp.say;
```

Comment:
@holli-holzer checkout the $*PROGRAM variable ;)

Reply:
TIMTOADY ^^